### PR TITLE
Update to work other kinds of projects

### PR DESCRIPTION
**Pull request by sgp667**
Hi @dokato I found your package today, really like it. 
I'd love to use this in my projects, but usually, I'm not working on new R package.

In my fork I basically added two functions that don't require the user to have all of their files in directories "R","tests" & "inst". Let me know if you'd be willing to integrate these changes into your project.
Summary of my changes:
 - updated todor function to search through all files in the working directory, created todor_package function to maintain original functionality of todor function
 - provided new RStudio addin function to account for refactoring of todor function
 - updated pattern searching(find_pattern function) to look for patterns just in comments( eg. hash before `TODO`)
 - removed find_pattern from namespace, it doesn't seem to serve any purpose

### CODE REVIEW

File: R/todor.R [hunk 1]
```diff
@@ -41,9 +40,33 @@ todor <- function(todo_types = NULL) {
   build_rstudio_markers(markers)
 }
 
+#' TODOR
```
Comment (dokato) [inline R/todor.R]: The function name is `Todor Package` now.

File: R/todor.R [hunk 2]
```diff
@@ -41,9 +40,33 @@ todor <- function(todo_types = NULL) {
...
+#'
+#' @param todo_types
+#'   todor that only checks directories specific to R packages
```
Comment (dokato) [inline R/todor.R]: Could you move the description up, and describe parameter?

File: R/todor.R [hunk 3]
```diff
@@ -41,9 +40,33 @@ todor <- function(todo_types = NULL) {
...
 #' Calls \code{todor} function.
-todor_addin <- function() {
-  todor()
+#'
+#' @export
+todor_project_addin <- function() {
+  todor(search_path = rstudioapi::getActiveProject())
+}
+
+#' Title
```
Comment (dokato) [inline R/todor.R]: Could you change the title to `Todor Package addin`?

**Comment by sgp667** [comment]
Sure, makes sense.
I updated the docs

**Comment by dokato** [comment]
Thank you very much. Now LGTM 👍

**Pull request merged**